Tools/setup/ubuntu.sh

```diff
@@ -194,6 +188,57 @@ if [[ $INSTALL_NUTTX == "true" ]]; then
 	fi
 fi
 
+# Install JAVA for SITL or FastRTPS
+if [[ $INSTALL_RTPS == "true" || $INSTALL_SIM == "true" ]]; then
+  if [[ "${UBUNTU_RELEASE}" == "18.04" ]]; then
+		java_version=11
+	else
+		java_version=14
+	fi
+
+  wget -O /tmp/jdk-public-key https://adoptopenjdk.jfrog.io/adoptopenjdk/api/gpg/key/public \
```

Comment:
The default packaged java doesn't work?

Reply:
I think the default one should work, agreed.

Reply:
I couldn't find an openjdk-14 repository for apt, if anything it would make this way easier, if you can find one please share the info.

This is the best I could find from Oracle [OpenJDK-14](https://jdk.java.net/14/) from what I can tell, they are only making the latest release available (17)

Reply:
On Ubuntu 20.04 there seems to be several different versions of openjdk packaged (11, 16, 17, etc)? 

Can we use any of those?